### ISSUE TITLE
Translate Terrible in Traditional Chinese

### ISSUE THREAD

**Issue by CookTurtle** [BDO-Planner/issues#261]
Hello This is a 4-year player from Taiwan Server.Traditional Chinese is our offical language.
In Traditional Chinese almost all UI translate are wrong!
It's looks like translate by Google it's a terrible idea.
But don't worry I can help BDOplanner update correct translate just like when I help BDOCodex translate correct
Last sorry for my bad English

**Comment by Shadowtrance** [comment]
Yes, sadly... pretty much all the interface text (so anything that isn't isn't class names, items etc which are from official sources) is google translate/other translators, translated, reverse translated and adjusted to hopefully make some sort of sense.
Which we basically said more or less in the 2.7.0 update notes.
Ihm and myself only speak english natively/fluently and also a little korean on my side too, so it's expected to not be perfect lol

@Ihellmasker can probably help further in getting you set up to fix the chinese translations if you want to help. :)

**Comment by Ihellmasker** [comment]
We use a tool called [Localazy](https://localazy.com/) for our translation. If you want to help improve our Chinese then I can invite you to that. I need your email address to invite you though. Could you send an email to me or add your email in here please so I can invite you. My email address is ihm@bdoplanner.com.

Thanks 👍🏻

**Comment by CookTurtle** [comment]
Send the email to Ihellmasker check it out

**Comment by CookTurtle** [comment]
Finish the translate!
